cli/pidexcd.go

```diff
@@ -0,0 +1,52 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"os"
+
+	"../cgroups"
+	"../docker"
+	log "github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+// pidExcdContainer returns containers whose process number exceeds a specified number
+func pidExcContainer(c *cli.Context) {
+	if len(c.Args()) != 1 {
+		log.Fatalf("Pidexc command takes exact one argument. See '%s pidexcd --help'.", c.App.Name)
+	}
+
+	excdNumStr := c.Args()[0]
+
+	excdNum, err := strconv.Atoi(excdNumStr)
+	if err != nil {
+		log.Fatalf("Pidexc command's argument should be positive, invalid input (%s)", excdNumStr)
```

Comment:
You said positive, while if the number is negative, you did not catch this situation.